tests/UnitConversionServiceTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+use FurnitureStoreApi\Services\UnitConversionService as UnitConversionService;
+class UnitConversionServiceTest extends \PHPUnit\Framework\TestCase
+{
+    public function testUnitConversionService_success()
+    {
+        $dimensions = new UnitConversionService();
```

Comment:
No need to create an object, just use the static method directly from the class

Reply:
Again, think I have done this correct now, although I'd like to double check I've done the right things for the malformed tests